deploy/airgap-prepare.sh

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ARCH=amd64
-SIDECAR_DOCKER_TAG=1.5.1
+SIDECAR_DOCKER_TAG=nightly
```

Comment:
Does this need to change to 1.6.0?

Reply:
The last information I got from Novus was that to keep that as nightly

Reply:
that will be addressed in another PR